### PR TITLE
Add .gitignore file

### PR DESCRIPTION
**Pull request by watilde**
I added `.gitignore` file from [the template](https://github.com/github/gitignore/blob/master/Node.gitignore) created by GitHub. Thanks :smiley_cat:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
```
Comment (ghost) [inline .gitignore:22]: Thanks! I forgot to add this as I usually keep this in my `.gitglobal`.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
```
Comment (ghost) [inline .gitignore]: I think we don't need this line...

**Comment by ghost** [comment]
Thanks! See comments. :+1:

File: .gitignore (hunk 2 above)
Comment (watilde) [inline .gitignore]: Ok, I remove it :+1:

**Comment by watilde** [comment]
@bucaran Thanks for your comments! I've updated and rebaseed it.

**Comment by ghost** [comment]
:+1: Awesome thanks @watilde

**Pull request merged**